docker/docker-compose-headless.yml

```diff
@@ -0,0 +1,14 @@
+version: '2.2'
```

Comment:
is docker-compose really necessary here? we are only using one container at a time. docker-compose is for launching groups of containers which work together.

Reply:
it's okay if it makes this more maintainable, but makefile+docker compose is starting to be a lot of layers.